tests/saw/s2n_handshake_io.saw

```diff
@@ -348,7 +348,8 @@ let yices_debug = do {yices;
 
 // Workaround for If then else on nat
 let equalNat_ite = core_axiom
```

Comment:
This file does not have a header - such a header might make the purpose of the file clearer.

Reply:
Agreed. I can add one.